tests/components/humidifier/common.py

```diff
@@ -0,0 +1,67 @@
+"""Collection of helper methods.
+
+All containing methods are legacy helpers that should not be used by new
+components. Instead call the service directly.
```

Comment:
This implies we should not add more of these helpers.
Call the service directly instead, we can add helpers in `tests/components/mqtt/test_humidifier.py` if it makes sense

Reply:
Removed the helper file. Added some helper functions in test_humidifier instead.